drivers/include/at86rf2xx.h

```diff
@@ -91,6 +91,18 @@ extern "C" {
 #   define RSSI_BASE_VAL                   (-91)
 #endif
 
+#if defined(DOXYGEN) || defined(MODULE_AT86RF232) || defined(MODULE_AT86RF233)
+/**
+ * @brief   Enable frame retry counter reporting
+ *
+ * The AT86RF2XX_HAVE_RETRIES flag enables support for NETOPT_TX_RETRIES NEEDED
+ * operation. Required for this functionality is the XAH_CTRL_2 register which
+ * contains the frame retry counter. Only the at86rf232 and the at86rf233
+ * support this register.
+ */
+#define AT86RF2XX_HAVE_RETRIES
```

Comment:
define this to 1 instead of just empty, makes it possible to check it in compiler code as well and not only with the preprocessor.

Reply:
doesn't an empty define implicitly define it to "1"?

Reply:
@kaspar030 from the command line, yes (`-DMYDEF` is the same as `-DMYDEF=1`), but I didn't think that was the case for the preprocessor defines.

Reply:
Please correct me if I am wrong

Reply:
I can't find any documentation to confirm @kaspar030 remark.

@gebart If you want to use the define in code as well, wouldn't it be necessary to define it as 0 by default and change it to 1 if a supported radio is enabled?

Reply:
@bergzand you're right, could you adjust this to have an `#else, #define AT86RF2XX_HAVE_RETRIES 0`, and change the `#ifdef` to `#if`?